src/services/UserService.ts

```diff
@@ -0,0 +1,37 @@
+import { Service } from '@tsed/common';
```

Comment:
I think the await keyword is a little abused in here. At one point, because everything is a Promise, it might come up with issues and race conditions.